lab/05-extend-aks-with-aci.md

```diff
@@ -0,0 +1,113 @@
+# Azure Kubernetes Service and Azure Container Instances
+
+Azure Container Instances (ACI) provide a hosted environment for running containers in Azure. When using ACI, there is no need to manage the underlying compute infrastructure, Azure handles this management for you. When running containers in ACI, you are charged by the second for each running container.
+
+When using the Virtual Kubelet provider for Azure Container Instances, both Linux and Windows containers can be scheduled on a container instance as if it is a standard Kubernetes node. This configuration allows you to take advantage of both the capabilities of Kubernetes and the management value and cost benefit of container instances.
+
+Install the ACI to Kubernetes connector using the az CLI:
+```
+az aks install-connector -n ignite18 -g ignite18
+```
+
+**Output:**
+```
+Merged "ignite18" as current context in /tmp/tmpbgwvxtlp
+Deploying the ACI connector for 'Linux' using Helm
+NAME:   aci-connector-linux-eastus
+LAST DEPLOYED: Thu Sep 20 17:59:05 2018NAMESPACE: defaultSTATUS: DEPLOYED
+
+RESOURCES:
+==> v1/Secret
+NAME                                                TYPE    DATA  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  Opaque  3     1s
+==> v1/ServiceAccount
+NAME                                                SECRETS  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  1        1s
+
+==> v1beta1/ClusterRoleBinding
+NAME                                                AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  1s
+
+==> v1beta1/Deployment
+NAME                                                DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks  1        1        1           0          1s
+
+==> v1/Pod(related)
+NAME                                                             READY  STATUS             RESTARTS  AGE
+aci-connector-linux-eastus-virtual-kubelet-for-aks-6c86769mzrt2  0/1    ContainerCreating  0         1s
+
+
+NOTES:
+The virtual kubelet is getting deployed on your cluster.
+
+To verify that virtual kubelet has started, run:
+
+  kubectl --namespace=default get pods -l "app=aci-connector-linux-eastus-virtual-kubelet-for-aks"
+
+Note:
+TLS key pair not provided for VK HTTP listener. A key pair was generated for you. This generated key pair is not suitable for production use.
+```
+
+The ACI connector automatically registers with the Kubernetes cluster as a virtual virtual. This virtual node can run Pods, just like the VM-based examples we've previously run.
+
+```
+kubectl get node
+```
+
+**Output:**
+```
+NAME                                         STATUS    ROLES     AGE       VERSION
+aks-nodepool1-25718494-0                     Ready     agent     1d        v1.11.2
+aks-nodepool1-25718494-1                     Ready     agent     1d        v1.11.2
+virtual-kubelet   Ready     agent     1m        v1.11.2
+```
+
+Let's deploy an application that schedules to our virtual node:
+
```

Comment:
here we know that you used a node selector to target the ACI node; it's probably important to stop, call out that you must "target" the node for scheduling IF you expect to run specific things on specific nodes...